rllab/envs/mujoco/gather/gather_env.py

```diff
@@ -401,9 +410,9 @@ def get_ori(self):
             obj = obj.wrapped_env
         try:
             return obj.get_ori()
-        except (NotImplementedError, AttributeError) as e:
+        except (NotImplementedError, AttributeError):
```

Comment:
If this is anything but completely meaningless, can you please at least omit a warning in the logger?

If the program cannot meaningfully continue when this error is raised, then do not suppress it.